apps/api/src/queries/buildViewModelForResource/viewModels/term.view-model.ts

```diff
@@ -31,6 +37,26 @@ import { CoscradDataExample } from '../../../test-data/utilities';
 import { DeepPartial } from '../../../types/DeepPartial';
 import { DTO } from '../../../types/DTO';
 
+class VocabularyListRecordForTerm {
```

Comment:
I can see a general pattern where we need a secondary view model for joins.

Reply:
I've now used this approach for the term view on the vocabulary list (entry) view.